object.js

```diff
@@ -0,0 +1,71 @@
+// 1. Hello object
+
+const user = {};
+
+user.name = "John";
+user.surname = "Smith";
+
+console.log(user);
+
+user.name = "Pete";
+
+console.log(user);
+
+delete user.name;
+
+console.log(user);
+
+
+// 2. Check for emptiness
+
+function isEmpty (obj){
+    return !Object.keys(obj)[0];
```

Comment:
if you went in this, just check the length of the array and not the first element

Reply:
done